src/api/Feed.js

```diff
@@ -445,7 +446,14 @@ class Feed extends Base {
         this.versionsAPI = new VersionsAPI(this.options);
 
         return new Promise(resolve => {
-            this.versionsAPI.getVersions(this.file.id, resolve, this.fetchFeedItemErrorCallback.bind(this, resolve));
+            this.versionsAPI.getVersions(
```

Comment:
How about a unit test for the `Feed.js` to capture the updated fields requested?